geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractPSetEXIntegrationTest.java

```diff
@@ -55,4 +56,17 @@ public void testPSetEX() {
 
     assertThat(jedis.pttl("key")).isGreaterThanOrEqualTo(1500);
   }
+
+  @Test
+  public void givenMoreThanFourArgumentsProvided_returnsWrongNumberOfArgumentsError() {
```

Comment:
This case is covered already by the `errors_givenWrongNumberOfArguments()` test, which uses the `assertExactNumberOfArgs()` method to verify that any number of arguments other than the expected (either too many or too few) results in a "ERR wrong number of arguments" message.